src/common/RillDeveloper.ts

```diff
@@ -0,0 +1,77 @@
+import { existsSync } from "fs";
+import type { RootConfig } from "./config/RootConfig";
+import { DuckDbConnection } from "./connection/DuckDbConnection";
+import type { DataModelerService } from "./data-modeler-service/DataModelerService";
+import type { DataModelerStateService } from "./data-modeler-state-service/DataModelerStateService";
+import { DataModelerStateSyncService } from "./data-modeler-state-service/sync-service/DataModelerStateSyncService";
+import type { MetricsService } from "./metrics-service/MetricsService";
+import type { NotificationService } from "./notifications/NotificationService";
+import { dataModelerServiceFactory } from "./serverFactory";
+
+/**
+ * Wrapper class that initializes other classes.
+ * To be used on the server only.
+ */
+export class RillDeveloper {
```

Comment:
As it stands this will only be used locally. Once we know for sure what we are going to do with the local and cloud versions we should do the refactor.

Reply:
This was something quick I put out. Note this wont work for client side instances. Adding the client support would have increased the scope of this PR even more. We can address generalisation when we have 2 different flavours of this.